week-4/Homework/mandatory/syllabus.js

```diff
@@ -0,0 +1,365 @@
+// let laptop = {
```

Comment:
Why is the code commented? Please remove comment of the code you want to be reviewed 😉